src/controller/CHIPDeviceController.cpp

```diff
@@ -740,6 +742,32 @@ CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, Re
     return err;
 }
 
+void DeviceCommissioner::OnDiscoveredDeviceOverBleSuccess(void * appState, BLE_CONNECTION_OBJECT connObj)
+{
+    auto self   = static_cast<DeviceCommissioner *>(appState);
+    auto device = self->mDeviceInPASEEstablishment;
+    if (nullptr != device)
+    {
+        auto remoteId = device->GetDeviceId();
+
+        RendezvousParameters params;
+        params.SetPeerAddress(Transport::PeerAddress::BLE()).SetConnectionObject(connObj);
```

Comment:
It almost works, but you're dropping the pincode which obviously causes SPAKE to fail. I did a quick test and added `.SetSetupPINCode(20202021)` to this line - with that addition it works.